.github/workflows/swift-toolchain.yml

```diff
@@ -179,6 +186,11 @@ jobs:
             echo CMAKE_Swift_FLAGS="-Xlinker -incremental:no -Xlinker -opt:ref -Xlinker -opt:icf" >> ${GITHUB_OUTPUT}
           fi
 
+          echo CMAKE_ANDROID_C_FLAGS="-ffunction-sections -fdata-sections -g" >> ${GITHUB_OUTPUT}
+          echo CMAKE_ANDROID_CXX_FLAGS="-ffunction-sections -fdata-sections -g" >> ${GITHUB_OUTPUT}
+          echo CMAKE_ANDROID_EXE_LINKER_FLAGS="" >> ${GITHUB_OUTPUT}
+          echo CMAKE_ANDROID_SHARED_LINKER_FLAGS="" >> ${GITHUB_OUTPUT}
```

Comment:
Should we consider sinking these into CMake caches or the swift build system?

Reply:
Makes sense to me if `-ffunction-sections` and -fdata-sections` are required to build for Android, and I imagine we would keep `-g` here?

Reply:
I need these across both the Runtime, Foundation, XCTest and other base projects, so I'm not sure it makes sense to sink these flags into some cmake config as we will need one in each project